### PR TITLE
chore: update package readme files [ci skip]

### PR DESCRIPTION
**Pull request by primeinteger**
## Purpose

I've updated most of the package's readme files (except the storybook-addon) to include a few details such as the NPM package name, a brief description, and link to the developer documentation.

I also turned off the wiki in the repo settings now that all of the wiki pages have been migrated to developer documentation site.

### CODE REVIEW

File: packages/components/README.md [hunk 1]
```diff
@@ -1,104 +1,7 @@
...
-);
-```
-
-You can also provide the tag name the container will use (which defaults to 'div'):
-
-```tsx
-{ wrapComponent: true, wrapContainerTag: 'span' }
-```
\ No newline at end of file
+Please refer to our [Documentation](https://www.contentful.com/developers/docs/experiences/) to learn more about it.
```
Comment (DanweDE) [inline packages/components/README.md:7]: Shouldn't we point to a more specific and relevant doc here? Maybe https://www.contentful.com/developers/docs/experiences/register-custom-components/#built-in-components

File: packages/components/README.md [hunk 2]
```diff
@@ -1,104 +1,7 @@
-# Experience Builder Components
+# @contentful/experiences-components-react
 
-> Experience Builder is currently in a private alpha and not available publicly. If you are interested in participating in the alpha, please reach out to your Contentful account team.
+This package provides the default structure and basic components for the Experiences SDK. You can use these components directly to start building your experiences, or as examples to create your own components.
```
Comment (DanweDE) [inline packages/components/README.md:3]: Let's also link somewhere for "Experiences SDK" considering a customer ends up here and wants to learn more? Maybe

```suggestion
This package provides the default structure and basic components for the [Experiences SDK](https://www.contentful.com/developers/docs/experiences/set-up-experiences-sdk/). You can use these components directly to start building your experiences, or as examples to create your own components.
```

or a link to the actual GH readme.

File: packages/components/README.md (hunk 1 above)
Comment (primeinteger) [reply packages/components/README.md]: Thanks for the feedback!  I agree and updated the link

File: packages/components/README.md (hunk 2 above)
Comment (primeinteger) [reply packages/components/README.md]: Yes, that's a good idea. Updated

File: packages/components/README.md [hunk 3]
```diff
@@ -1,104 +1,7 @@
...
-
-## Components
-
-The following components are available:
-
-- [Button](src/components/Button/README.md)
-- [Heading](src/components/Heading/README.md)
-- [Image](src/components/Image/README.md)
-- [RichText](src/components/RichText/README.md)
-- [Text](src/components/Text/README.md)
```
[the remark this answers is not in the document]
Comment (primeinteger) [reply packages/components/README.md]: Sounds good. I restored this content

**Review by DanweDE — APPROVED** [review]

**Pull request merged**